batch_seqr_loader/utils.py

```diff
@@ -19,7 +19,7 @@
 
 
 AR_REPO = 'australia-southeast1-docker.pkg.dev/cpg-common/images'
-GATK_VERSION = '4.2.1.0'
+GATK_VERSION = '4.2.2.0-cpgfix00'
```

Comment:
What's that `cpgfix00`? ;)

Reply:
Our fork with a couple of fixes:
https://github.com/populationgenomics/gatk/commits/master

Reported one of them back to GATK, hoping to get back to the official release at some point :) 
https://github.com/broadinstitute/gatk/issues/7483

For now, building this image:
https://github.com/populationgenomics/production-pipelines/tree/initial/dockers/gatk

I'll add a comment to the code!

Reply:
Oh, nice! Did you also submit a PR to the GATK repo?